featurecontext/init.go

```diff
@@ -0,0 +1,34 @@
+package featurecontext
+
+import (
+	"crypto/tls"
+	"fmt"
+	"net/http"
+	"time"
+
+	providerv1beta1 "github.com/cs3org/go-cs3apis/cs3/storage/provider/v1beta1"
+	"github.com/cs3org/reva/pkg/rgrpc/todo/pool"
+)
+
+func (f *FeatureContext) Init(endpoint string, httpInsecure bool) {
+
+	client, err := pool.GetGatewayServiceClient(endpoint)
+	if err != nil {
+		// how to handle this?
+		fmt.Println("couldnt open a connection")
```

Comment:
This probably shouldn't continue without a connection.

Reply:
added logging a fatal error to exit the program